x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_upgrade_without_preview.md

```diff
@@ -798,3 +808,166 @@ And user opens the Rule Updates table
 Then user should see prebuilt rules available to upgrade
 But user should not be able to upgrade them
 ```
+
+### Licensing
+
+#### **Scenario: Prebuilt rule is upgraded to target version when upgraded via upgrade table on insufficient license**
+
+**Automation**: 1 e2e test with a mock rule.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And an installed prebuilt rule
+And a base version exists for this rule
+And this rule is outdated (a new version is available for this rule)
+And this rule is <customization_state>
+When user is on the Rule Updates table
+And clicks on upgrade button for this rule
+Then success message should be displayed after upgrade
+And the upgraded prebuilt rule should be removed from the table
+And all customizable rule fields should be equal to the target version
+
+<customization_state> = customized | not customized
+```
+
+#### **Scenario: Multiple selected rules are upgraded to target versions when upgraded via upgrade table on insufficient license**
+
+**Automation**: 1 e2e test with a mock rule.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And multiple outdated prebuilt rules (new versions are available for them)
+When user is on the Rule Updates table
+And selects multiple rules
+And clicks a button to upgrade selected rules
+Then success message should be displayed after upgrade
+And all customizable fields of each upgraded rule should be equal to their respective target versions
+```
+
+#### **Scenario: All rules are upgraded to target versions when upgraded via upgrade table on insufficient license**
+
+**Automation**: 1 e2e test with a mock rule.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And multiple outdated prebuilt rules (a new version is available for them)
+When user is on the Rule Updates table
+And upgrades all rules at once using "Upgrade all" button
+Then success message should be displayed after upgrade
+And all customizable fields of each upgraded rule should be equal to their respective target versions
+```
```

Comment:
All these 3 scenarios are the same with the exception what button the user clicked. It should be simpler to describe one scenario and elaborate the differences via `Examples`.